content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -313,16 +313,11 @@ limit, the same as the scheduler.
 A Pod can restart, causing re-execution of init containers, for the following
 reasons:
 
-* A user updates the Pod specification, causing the init container image to change.
-  Any changes to the init container image restarts the Pod. App container image
-  changes only restart the app container.
 * The Pod infrastructure container is restarted. This is uncommon and would
   have to be done by someone with root access to nodes.
-* All containers in a Pod are terminated while `restartPolicy` is set to Always,
-  forcing a restart, and the init container completion record has been lost due
-  to garbage collection.
-
 
+Since 1.20, The Pod will not be restarted when the init container image is changed,
```

Comment:
I'd phrase this more like, say:
This applies for Kubernetes v1.20 and later. If you are using an earlier version of Kubernetes,
consult the documentation for the version you are using.

Does that seem reasonable?

Reply:
Updated. Thanks for the comment.